test/tt_store_suite.erl

```diff
@@ -0,0 +1,32 @@
+-module(tt_store_suite).
+-include_lib("common_test/include/ct.hrl").
+
+-export([all/0]).
+-export([test0/1, test1/1, test2/1, test3/1]).
+
+all() -> [test0, test1, test2].
+
+test0(_) ->
+    tt_store:start_link(),
+    tt_store:add("Bum", {10,0,0}, {11,0,0}, "Bah"),
+    Res = tt_store:find_by_time({10,0,0}, {11,0,0}),
+    Res = [{talk,"Bum",{10,0,0},{11,0,0},"Bah"}].
```

Comment:
Use [EUnit](http://www.erlang.org/doc/apps/eunit/chapter.html) assertions in tests:

```erlang
 -include_lib("eunit/include/eunit.hrl").
...
test0(_) ->
     tt_store:start_link(),
     tt_store:add("Bum", {10,0,0}, {11,0,0}, "Bah"),
     Res = tt_store:find_by_time({10,0,0}, {11,0,0}),
     ?assertMatch([{talk,"Bum",{10,0,0},{11,0,0},"Bah"}], Rest).
```